homeworks/ovarikristof/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,14 @@
+full_name = (input("Please enter your full name."))         #BEKÉRJÜK A TELJES NEVET.
+
+age = (str(input("Please enter your age.")))                #BEKÉRJÜK A KORÁT.
+
+experince_in_python = (str(input("Please enter your experience with python in years.")))    #BEKÉRJÜK, HOGY HÁNY ÉV TAPSZTALATA VAN PYTHON-BAN.
+
+age_in_days = int((age)) * 365              #MEGHATÁROZZUK, HOGY A MEGADOTT KOR AZ MENNYI NAPOKBAN KIFEJEZVE.
+
+
+
+print("My character's name is " + (full_name.title().strip()) + ".")
```

Comment:
Tanultunk az f-stringekről, nem muszáj megváltoztatni, de jóval elegánsabb megoldás mint a concatenation (+ ok használata összefűzésre).
Ezen felül tudsz több sort kiprintelni egyetlen egy print() hívással, valahogy így:
print("""minden
amit
ki akarsz
írni""")

Természetesen itt is tudsz f-stringet használni, csak írj az elejére egy f betűt:
f"""
"""